ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -258,19 +258,19 @@ def check_for_halted_volumes(logger):
                     except (ClusterNotReachableException, RuntimeError) as ex:
                         logger.failure(
                             "Seems like the Volumedriver {0} is not running.".format(vp.name, ex.message),
-                            'halted_{0}'.format(vp.name))
+                            'halted_volumedriver_{0}'.format(vp.name))
```

Comment:
The success call is on line 273: https://github.com/openvstorage/openvstorage-health-check/pull/216/files#diff-9fb892c435d6b67601bb1471272037a1R273